OrcanodeMonitor/Models/OrcanodeEvent.cs

```diff
@@ -69,6 +69,7 @@ public OrcanodeEvent(Orcanode node, string type, string value, DateTime timestam
             Value = value;
             DateTimeUtc = timestamp;
             OrcanodeId = node.ID;
+            year = DateTime.UtcNow.Year.ToString();
```

Comment:
```suggestion
            Year = DateTime.UtcNow.Year.ToString();
```
capitalize for consistency with the existing fields